yarn.lock

```diff
@@ -4864,7 +4864,7 @@ lodash.toarray@^4.4.0:
   resolved "https://registry.yarnpkg.com/lodash.toarray/-/lodash.toarray-4.4.0.tgz#24c4bfcd6b2fba38bfd0594db1179d8e9b656561"
   integrity sha1-JMS/zWsvuji/0FlNsRedjptlZWE=
 
-lodash@^4.17.10, lodash@^4.17.11, lodash@^4.17.14, lodash@^4.17.19, lodash@^4.17.20, lodash@^4.17.21:
+lodash@>=4.17.21, lodash@^4.17.10, lodash@^4.17.11, lodash@^4.17.14, lodash@^4.17.19, lodash@^4.17.20, lodash@^4.17.21:
```

Comment:
The following error is consistently thrown unless the `yarn.lock` file is updated. Not sure why this is happening...

```
profile.ts:106 Uncaught TypeError: Cannot read property 'updateActiveProfile' of undefined
    at profile.ts:106
    at Object.subscribe (index.mjs:51)
    at Module.../shared/lib/profile.ts (profile.ts:105)
    at __webpack_require__ (bootstrap:21)
    at fn (hot module replacement:61)
    at Module.../shared/lib/app.ts (index.js:255200)
    at __webpack_require__ (bootstrap:21)
    at fn (hot module replacement:61)
    at Module.../shared/lib/router.ts (index.js:258905)
    at __webpack_require__ (bootstrap:21)
```